CODEOWNERS

```diff
@@ -0,0 +1,143 @@
+lib/vdsm/virt/                          @mz-pdm
+tests/virt/                             @mz-pdm
+tests/virt/livemerge_test.py            @nirs
+
+# Network
+lib/vdsm/network/                       @almusil
+pyproject.toml                          @almusil
+tests/network                           @almusil
+
+# Gluster - maintainer needed
+lib/vdsm/gluster/
+
+# Infra
+lib/vdsm/alignmentScan.py               @nirs @vjuranek
+lib/vdsm/api/                           @tinez
+lib/vdsm/clientIF.py                    @mz-pdm @nirs @vjuranek
+lib/vdsm/client.py                      @tinez
+lib/vdsm/config.py                      @tinez
+lib/vdsm/constants.py                   @tinez
+lib/vdsm/cpuinfo.py                     @mz-pdm
+lib/vdsm/dmidecodeUtil.py               @mz-pdm
+lib/vdsm/executor.py                    @nirs @mz-pdm
+lib/vdsm/health.py                      @nirs
+lib/vdsm/hook/                          @tinez
+lib/vdsm/host/                          @tinez
+lib/vdsm/hugepages.py                   @mz-pdm
+lib/vdsm/jobs.py                        @nirs @vjuranek @mz-pdm
+lib/vdsm/jsonrpcvdscli.py               @tinez
+lib/vdsm/kvm2ovirt.py                   @mz-pdm
+lib/vdsm/logUtils.py                    @nirs @mz-pdm @tinez
+lib/vdsm/machinetype.py                 @mz-pdm
+lib/vdsm/metrics/                       @tinez
+lib/vdsm/mkimage.py                     @mz-pdm
+lib/vdsm/moduleloader.py                @tinez
+lib/vdsm/momIF.py                       @mz-pdm
+lib/vdsm/numa.py                        @mz-pdm
+lib/vdsm/osinfo.py                      @tinez
+lib/vdsm/ppc64HardwareInfo.py           @mz-pdm
+lib/vdsm/protocoldetector.py            @tinez
+lib/vdsm/rpc/                           @tinez
+lib/vdsm/schedule.py                    @nirs @mz-pdm
+lib/vdsm/sslutils.py                    @tinez
+lib/vdsm/supervdsm_server.py            @tinez
+lib/vdsm/taskset.py                     @nirs
+lib/vdsm/throttledlog.py                @mz-pdm
+lib/vdsm/tool/                          @tinez
+lib/vdsm/utils.py                       @tinez
+lib/vdsm/v2v.py                         @mz-pdm
+lib/vdsm/vdsm.conf.sample               @tinez
+lib/vdsm/vdsmd.py                       @tinez
+lib/vdsm/virtsparsify.py                @mz-pdm
+lib/vdsm/virtsysprep.py                 @mz-pdm
+
+# Profiling
+lib/vdsm/profiling/                     @nirs
+lib/vdsm/profiling/memory
+
+# Common
+lib/vdsm/common/                        @tinez
+lib/vdsm/common/api.py                  @tinez @nirs
+lib/vdsm/common/base26.py               @mz-pdm
+lib/vdsm/common/cache.py                @nirs @tinez
+lib/vdsm/common/cmdutils.py             @tinez @nirs
+lib/vdsm/common/commands.py             @tinez @nirs
+lib/vdsm/common/compat.py               @tinez @nirs
+lib/vdsm/common/concurrent.py           @nirs
+lib/vdsm/common/config.py.in            @tinez
+lib/vdsm/common/constants.py.in         @tinez
+lib/vdsm/common/conv.py                 @mz-pdm
```

Comment:
Well, owning this should motivate me to remove this ugliness one day, right? :-)